python/lsst/cp/pipe/ptc.py

```diff
@@ -922,14 +948,11 @@ def errFunc(p, x, y):
             varVecOriginal = np.array(dataset.rawVars[ampName])
             varVecOriginal = self._makeZeroSafe(varVecOriginal)
 
-            mask = ((meanVecOriginal >= self.config.minMeanSignal) &
```

Comment:
This mask is no longer needed because of the filtering on line 346?  Was this mask unnecessary in the previous version of the code?  I'm a bit confused why this isn't necessary (likely because I haven't read through the call stack again).

Reply:
Right. Before, this was at this point (the EXPAPPROX and POLYNOMIAL path of the code). Than I moved it to 346, so now it applies to both paths of the code (FULLCOV being the other one). so, leaving it here would be redundant.